src/cpp/src/utils.hpp

```diff
@@ -70,6 +70,28 @@ ov::genai::StreamerVariant get_streamer_from_map(const ov::AnyMap& config_map);
 
 ov::genai::OptionalGenerationConfig get_config_from_map(const ov::AnyMap& config_map);
 
+// https://gist.github.com/lorenzoriano/5414671
+template <typename T, typename U>
+std::vector<T> linspace(U start, U end, size_t num, bool endpoint = false) {
```

Comment:
should be kept in `text2image` utils? I'm not sure that it can be useful in general for GenAI for now.. if it will be in future, we will have a common header with numpy helpers.

Reply:
There is no utils file in [text2image ](https://github.com/openvinotoolkit/openvino.genai/tree/master/src/cpp/src/text2image) folder. Should I add a new file?

Reply:
yes

Reply:
Isn't that just std::iota()?

Reply:
done